sdk/schemaregistry/azure-data-schemaregistry/swagger/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Schema Registry for Java
+
+> see https://aka.ms/autorest
```

Comment:
Should this be populated with schema-registry docs?

Reply:
The swagger/README is different in that autorest reads generation instructions from it. I see most of the suppressions for that README format in: eng/.docsettings.yml

Example: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/search/azure-search-documents/swagger/README.md